.github/workflows/windows.yml

```diff
@@ -47,6 +47,13 @@ jobs:
     - name: Enter DevShell
       run: 'libva-utils\.github\workflows\EnterDevShell.ps1 ${{ inputs.architecture }}'
       shell: pwsh
+    - name: Build DirectX-Headers

```

Comment:
oh, so that's official github project which needs to be built as a prerequisite. I suggest you add a note about it and link in the meson.build of libva. I personally was not aware of this project existence and suspect that many people with linux experience won't be aware as well. So, link might be helpful to fast identify what's needed.

Reply:
To remove the need to build/install the dependency manually, with the latest pushed changes, meson now will automatically resolve the dependency by cloning the `DirectX-Headers` repo as part of the meson build configuration step if the `directx-headers` dependency is not found on the building system.